### DISCUSSION TITLE
Elsa 3 Custom Activity Create Outcomes

### DISCUSSION THREAD

**Discussion by GaryJurgens** [elsa-workflows/elsa-core#4204] [Q&A] [answered]
hi

I am trying to rewrite my Customs activities from Elsa 2 to Elsa 3, however, I am confused on how to Create Outcomes such as Success / Fail like you could define in Elsa 2, I know Elsa 3 is a Complete Rewrite. The outcomes Section of the Documentation is Missing / not completed yet. 

in Elsa 2 you could define your Outcomes like the following.

`[Activity(
       Category = "FormFlow",
       DisplayName = "Send SMS/ Text message",
       Description = "Send SMS / Text message, Requires JSON String from HTTP Endpoint JSON Serialized from Type : SmsMessageModel. ",
       Outcomes = new[] { "SMS_Sent", "SMS_Failed",  }

   )]`

how would you do this in Elsa 3 ?

thanks G

**Comment by sfmskywalker** [comment, marked as answer]
Try this:

```csharp
[Activity("MyNamespace", "FormFlow", "Send SMS / Text message, Requires JSON String from HTTP Endpoint JSON Serialized from Type : SmsMessageModel.", DisplayName = "Send SMS/ Text message"]
[FlowNode("SMS_Sent", "SMS_Failed")]
```

**Reply by GaryJurgens** [reply]
ah thanks a Million,  would never have have worked that one out.
